### PR TITLE
Collected new core keys

### PR DESCRIPTION
**Pull request by mauthi**

### CODE REVIEW

**Comment by mauthi** [comment]
#29

File: src/resources/lang/enso/ro.json [hunk 1]
```diff
@@ -1,6 +1,151 @@
 {
-    "Key": null,
+    "Die Aktion war erfolgreich": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: ?

File: src/resources/lang/enso/ro.json [hunk 2]
```diff
@@ -1,6 +1,151 @@
...
+    "App": null,
     "Value": null,
+    "Key": null,
+    "Core": null,
+    "Only missing": null,
+    "edit texts": null,
+    "keys found": null,
+    "German": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: where is this from?

File: src/resources/lang/enso/ro.json [hunk 3]
```diff
@@ -1,6 +1,151 @@
...
+    "data import": null,
+    "The tutorial was created!": null,
+    "Permission": null,
+    "Tutorials Index": null,
+    "The log was cleaned": null,
+    "The log file": null,
+    "Current file size is": null,
+    "was last updated": null,
+    "Show Log": null,
+    "MB": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json:39]: ?

File: src/resources/lang/enso/ro.json [hunk 4]
```diff
@@ -1,6 +1,151 @@
...
+    "Last updated": null,
+    "Size": null,
+    "The role was created!": null,
+    "Roles Index": null,
+    "The menu was created!": null,
+    "Menus Index": null,
+    "The permissions were created!": null,
+    ".selectOptions": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: we don't want starting from here

File: src/resources/lang/enso/ro.json [hunk 5]
```diff
@@ -1,6 +1,151 @@
...
+    ".getTableData": null,
+    ".destroy": null,
+    ".initTable": null,
+    ".edit": null,
+    ".update": null,
+    ".show": null,
+    ".store": null,
+    "Resource Prefix": null,
+    ".create": null,
+    ".index": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: to here, except "Resource Prefix"

File: src/resources/lang/enso/ro.json [hunk 6]
```diff
@@ -1,6 +1,151 @@
...
+    "resource": null,
+    "Create Resource": null,
+    "Operation failed because the permission is allocated to existing role(s)": null,
+    "records": null,
+    "total": null,
+    "filtered": null,
+    "from": null,
+    "The permission was created!": null,
+    "Permission Group": null,
+    "Lesen": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: ?

File: src/resources/lang/enso/ro.json [hunk 7]
```diff
@@ -1,6 +1,151 @@
...
+    "Schreiben": null,
```
Comment (aocneanu) [inline src/resources/lang/enso/ro.json]: ?

**Review by aocneanu — COMMENTED** [review]
There are some keys in German, I can't identify where they are from (because I don't speak the language:) )

File: src/resources/lang/enso/ro.json (hunk 1 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: removed

File: src/resources/lang/enso/ro.json (hunk 2 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: removed

File: src/resources/lang/enso/ro.json (hunk 3 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json:39]: not from me and not german - I think it's megabyte?

File: src/resources/lang/enso/ro.json (hunk 4 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: ?

Comment (mauthi) [reply src/resources/lang/enso/ro.json]: I remove the .xxx again but if we collect again they will be added again

File: src/resources/lang/enso/ro.json (hunk 5 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: done

File: src/resources/lang/enso/ro.json (hunk 6 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: removed

File: src/resources/lang/enso/ro.json (hunk 3 above)
Comment (aocneanu) [reply src/resources/lang/enso/ro.json:39]: I think you're right, somewhere in the logs index / show

File: src/resources/lang/enso/ro.json (hunk 7 above)
Comment (mauthi) [reply src/resources/lang/enso/ro.json]: removed

File: src/resources/lang/enso/ro.json (hunk 4 above)
Comment (aocneanu) [reply src/resources/lang/enso/ro.json]: this is what I meant by "the flag shoud be used with care" :)

**Comment by mauthi** [comment]
made the updates and removed german keys

**Comment by aocneanu** [comment]
350+ per language.... and we though that we had localisation :)))

**Comment by mauthi** [comment]
not 350+
the 350+ are because we store the keys now with linebreaks
but about 250+ :)

**Pull request merged**